### ISSUE TITLE
trailing slash in repo1-path for S3 problem

### ISSUE THREAD

**Issue by marco44** [pgbackrest/pgbackrest#691]
Please provide the following information when submitting an issue (feature requests or general comments can skip this):

1. pgBackRest version: 2.10-1.pgdg90+1

2. PostgreSQL version: PostgreSQL 10.6 (Debian 10.6-1.pgdg90+1)

3. Debian stretch (10)

4. pgBackRest from PGDG package

I have been dealing with a simple but annoying bug for the last 2 hours, using a S3 repo...
If I have a trailing / in the repo1-path, such as:
```
repo1-path=/pgbackrest/
```
creating stanzas, making backups, archiving, etc… all this works. But info returns:
```
pgbackrest --config /var/lib/postgresql/pgbackrest.conf --stanza=10-qualif-main info
No stanzas exist in the repository.
```
Removing the / solves the problem. But it would be easier if either pgbackrest removed the /, or raised an error.

Regards.

**Comment by dwsteele** [comment]
Yeah, that's no good.  We'll fix it.

I'm guessing this is an issue in the new C code, since the test of those commands are still in Perl.

**Comment by dwsteele** [comment]
Fixed at 9382283586da4.  Sorry for the noise.
